### ISSUE TITLE
ModuleNotFoundError: No module named 'AutoDockTools.Utilities24.prepare_dpf4'

### ISSUE THREAD

**Issue by iwatobipen** [Valdes-Tresanco-MS/AutoDockTools_py3#8]
Dear developer,

Thanks for sharing the code.
There isn't prepare_dpf4.py in Utilities24 directory but prepare_dpf42.py is.
Is it typo of setup.py?
>        'prepare_dpf4 = AutoDockTools.Utilities24.prepare_dpf4:main',
Thanks,

**Comment by arunppsg** [comment]
I had a similar error. `prepare_dpf4` got installed and when I ran `prepare_dpf4 -v` from the terminal, it throwed the following error:
```py
Traceback (most recent call last):
  File "/home/arun/miniconda3/bin/prepare_dpf4", line 5, in <module>
    from AutoDockTools.Utilities24.prepare_dpf4 import main
ModuleNotFoundError: No module named 'AutoDockTools.Utilities24.prepare_dpf4'
```
A hacky fix was find the path of
1. `prepare_dpf4` (`which prepare_dpf4`)
2. Open it in the editor and update this import statement `from AutoDockTools.Utilities24.prepare_dpf4 import main` as `from AutoDockTools.Utilities24.prepare_dpf42 import main`
3. Rename the file `prepare_dpf4` as `prepare_dpf42`

**Comment by Valdes-Tresanco-MS** [comment]
Fixed #10
